### DISCUSSION TITLE
Does Atlantis PrimeNG support RTL?

### DISCUSSION THREAD

**Discussion by khaled6mmd** [primefaces/community#801] [PrimeNG Templates] [answered]
Does Atlantis - PrimeNG support RTL if does not is there another solution?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Atlantis doesn't have RTL support unfortunately. Templates that support RTL are:

 - serenity 
 - roma 
 - poseidon

**Reply by maqadeersaeed** [reply]
What About Ultima??? We were using Ultima 13 that had RTL Support. But I am unable to see it in Ultima 17

**Reply by sebwastiau** [reply]
@cetincakiroglu those templates are compatible with Angular 17?

**Reply by cetincakiroglu** [reply]
@maqadeersaeed 

We removed RTL support in v17 from Ultima because RTL feature was custom-added with CSS and it was a feature that did not have any impact on the Prime components. In the future, we plan to provide RTL support for Prime UI Libraries and also improve RTL support for templates.

**Reply by cetincakiroglu** [reply]
@sebwastiau 

Yes, they are except serenity. But as a side note, we've removed RTL option from the v17 templates. You can find the explanation of it in the comment above.

**Reply by AshishFreak** [reply]
Hi @cetincakiroglu sir please reply can you please give me poseidon theme
